tests/integration/test_contact_template.py

```
@@ -729,6 +809,206 @@ def test_precomputation_stores_aroi_domain_for_vanity_urls(self):
+            output_dir=self.temp_dir,
+            onionoo_url="https://test.example.com",
+            relay_data=self.relay_data,  # Original test data: 1 relay per contact
+            use_bits=False,
+            progress=False,
+            mp_workers=0,
+        )
+
+        relay_set.write_pages_by_key("contact")
+
+        contact_hashes = list(relay_set.json["sorted"]["contact"].keys())
+        self.assertGreater(len(contact_hashes), 0)
```

Ruff (0.15.5)

[warning] Use a regular `assert` instead of unittest-style `assertGreater`

Replace `assertGreater(...)` with `assert ...`

(PT009)